### DISCUSSION TITLE
issue with leaking launcher and uncertain if it's a know issue and if not, where to report it

### DISCUSSION THREAD

**Discussion by azatoth** [paketo-buildpacks/feedback#286] [General]
In https://github.com/paketo-buildpacks/health-checker/issues/87 there's a issue that `/cnb/lifecycle/launcher` is leaking spring-boot initialization, which is most noticeable when the `health-check` is ran and it will try to allocate 2GB memory (when it is processing all the certificates).

I don't think it's directly related to health-checker, but something else, but I don't know which entity might be responsible so I ask if someone can help me here?

**Comment by dmikusa** [comment]
See https://github.com/paketo-buildpacks/health-checker/issues/87#issuecomment-2129523234
